easybuild/easyconfigs/p/Phantompeakqualtools/Phantompeakqualtools-1.2.2-foss-2021b.eb

```
@@ -0,0 +1,54 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'Tarball'
+
+name = 'Phantompeakqualtools'
+version = '1.2.2'
+
+homepage = "https://github.com/kundajelab/phantompeakqualtools"
+
+description = """ It computes informative enrichment and quality measures for ChIP-seq/DNase-seq/FAIRE-seq/MNase-seq data."""
```

The blank line can be removed, and the description line is a bit too long:
```suggestion
description = """It computes informative enrichment and quality measures
for ChIP-seq/DNase-seq/FAIRE-seq/MNase-seq data."""
```